src/components/Collections/CollectionOptions/QuickOptions.tsx

```diff
@@ -25,6 +25,7 @@ export function QuickOptions({
             {!(
                 collectionSummaryType === CollectionSummaryType.trash ||
```

Comment:
not actually related to this but we should ideally write this conditional statements as

```
collectionSummaryType === CollectionSummaryType.trash ?
<TrashQuickOptions>:
collectionSummaryType === CollectionSummaryType.uncategoried?
<UncategoriedQuickOptions>:
...
```

That way the code will be more cleaner